get-substrate.sh

```diff
@@ -49,15 +161,50 @@ else
 	echo "Please refer to https://github.com/paritytech/substrate for setup information."
 fi
 
-if ! which rustup >/dev/null 2>&1; then
-	curl https://sh.rustup.rs -sSf | sh -s -- -y
-	source ~/.cargo/env
-else
+APP="Rust"
+if ! rustup 2>&1 | grep "command not found"; then
+	echo "Updating $APP ..."
 	rustup update
+else
+	echo "Installing $APP ..."
+	curl https://sh.rustup.rs -sSf | sh -s -- -y;
+	source ~/.cargo/env
+fi
+echo "Switching to $APP Stable";
+rustup default stable;
+
+echo "Installing WASM dependencies"
+rustup target add wasm32-unknown-unknown --toolchain nightly;
+cargo install --force --git https://github.com/alexcrichton/wasm-gc;
+cargo install --force --git https://github.com/pepyakin/wasm-export-table.git;
+
+if which substrate 2>&1 | grep ".cargo/bin/substrate"; then
+	EXISTING_SUBSTRATE_VERSION=$(substrate --version)
+	echo -e "Substrate version $EXISTING_SUBSTRATE_VERSION detected on host machine"
 fi
-cargo install --force --git https://github.com/paritytech/substrate subkey
-cargo install --force --git https://github.com/paritytech/substrate substrate
 
+f=`mktemp -d`
+git clone https://github.com/paritytech/substrate-node-template $f
+# fetch new tags from remote repo
+git fetch --tags
```

Comment:
Which tags are you fetching here?

Reply:
I finally understand what you mean. I was meant to be fetching the tags of the substrate repo, not the substrate-node-template repo :)